### PR TITLE
Add conversion for Case

### PR DESCRIPTION
**Pull request by vserret**

### CODE REVIEW

File: src/expr/identifier.rs [hunk 1]
```diff
@@ -206,7 +206,7 @@ mod tests {
         println!("head = {}", id.head().unwrap());
         println!("tail = {}", id.tail().unwrap());
         println!("tail^2 = {}", id.tail().unwrap().tail().unwrap());
-        assert!(id.tail().unwrap().tail().unwrap().tail().unwrap().len() == 2)
```
[the remark this answers is not in the document]
Comment (ngrislain) [reply src/expr/identifier.rs:209]: No it is working

[the remark this answers is not in the document]
Comment (ngrislain) [reply src/expr/identifier.rs:209]: Can you revert this please?

File: src/sql/mod.rs [hunk 2]
```diff
@@ -167,6 +167,9 @@ mod tests {
         WITH t1 AS (SELECT a,d FROM table_1),
         t2 AS (SELECT * FROM table_2)
         SELECT * FROM t1 INNER JOIN t2 ON t1.d = t2.x INNER JOIN table_2 ON t1.d=table_2.x ORDER BY t1.a, t2.x, t2.y, t2.z LIMIT 17",
+        "SELECT CASE a WHEN 5 THEN 0 ELSE a END FROM table_1",
+        "SELECT CASE WHEN a < 5 THEN 0 WHEN a < 3 THEN 3 ELSE a END FROM table_1",
+        //"SELECT CASE WHEN a < 5 THEN 0 WHEN a < 3 THEN 3 END FROM table_1",
```
Comment (ngrislain) [inline src/sql/mod.rs:172]: Why do we have this commented?

Comment (vserret) [reply src/sql/mod.rs:172]: Because for the last one we need to implement Value::Null first

File: tests/integration.rs [hunk 3]
```diff
@@ -62,6 +62,9 @@ const QUERIES: &[&str] = &[
     WITH t1 AS (SELECT a,d FROM table_1),
     t2 AS (SELECT * FROM table_2)
     SELECT * FROM t1 INNER JOIN t2 ON t1.d = t2.x INNER JOIN table_2 ON t1.d=table_2.x ORDER BY t1.a, t2.x, t2.y, t2.z LIMIT 17",
+    "SELECT CASE a WHEN 5 THEN 0 ELSE a END FROM table_1",
+    "SELECT CASE WHEN a < 5 THEN 0 WHEN a < 3 THEN 3 ELSE a END FROM table_1",
+    //"SELECT CASE WHEN a < 5 THEN 0 WHEN a < 3 THEN 3 END FROM table_1",
```
Comment (ngrislain) [inline tests/integration.rs:67]: Can you fix this please?

**Review by ngrislain — APPROVED** [review]

**Pull request merged**